rush-plugins/rush-azure-storage-build-cache-plugin/src/AzureStorageBuildCacheProvider.ts

```diff
@@ -3,11 +3,14 @@
 
 import { ITerminal } from '@rushstack/node-core-library';
 import { PrintUtilities } from '@rushstack/terminal';
-import type {
+import {
   ICloudBuildCacheProvider,
-  ICredentialCache,
+  CredentialCache,
   ICredentialCacheEntry,
-  RushSession
+  RushSession,
```

Comment:
Since All API comes from `rush-sdk`, it seems `RushSession` related stuff can be totally removed in this file now.

Reply:
Let's keep it as a formalism, because generally we DO want to move plugin context onto the session and move away from global state.  As one example, I suggested that maybe `CredentialCache` should eventually be created+disposed by Rush and obtained via `RushSession`.